optuna/notebooks/jupytercon_demo/RAPIDS_xfeat_Optuna-GPU.ipynb

```diff
@@ -906,149 +925,76 @@
     "    df_[col] = df_[col].astype(\"object\")\n",
     "\n",
     "with timed(\"Feature Engineering\"):\n",
-    "    df_feature_eng = feature_engineering(df_)\n",
-    "    df_feature_eng[TARGET_COL] = df_feature_eng[TARGET_COL].astype('float64')\n",
-    "    score = train_and_eval(df_feature_eng)\n",
+    "    df_feature_eng_train, df_feature_eng_test = feature_engineering(df_train, df_test)\n",
+    "    df_feature_eng_train[TARGET_COL] = df_feature_eng_train[TARGET_COL].astype('float64')\n",
```

Comment:
Why cast to float64? We should be all float32 right?

Reply:
It needs to be float but either of them works fine, I'll make it float32